crates/ilp-cli/src/main.rs

```diff
@@ -0,0 +1,187 @@
+use clap::{crate_version, App, Arg, ArgMatches, SubCommand};
+use reqwest;
+use std::{collections::HashMap, process::exit};
+
+pub fn main() {
+    // Define the arguments to the CLI
+    let mut app = App::new("ilp-cli")
+        .about("Interledger.rs Command-Line Interface")
+        .version(crate_version!())
+        // TODO remove this line once this issue is solved:
+        // https://github.com/clap-rs/clap/issues/1536
+        .after_help("")
+        .args(&[
+            Arg::with_name("authorization_key")
+                .long("auth")
+                .env("ILP_CLI_AUTH")
+                .required(true)
+                .help("An authorization key granting access to the designated operation"),
+            Arg::with_name("node_address")
```

Comment:
In other places we call the URL a client is going to connect to `_url` and the address + port a service is going to bind to `_bind_address`
```suggestion
            Arg::with_name("node_url")
```